### PR TITLE
Update sbert_rerank.py: added "device" parameter to SentenceTransform…

### PR DESCRIPTION
**Pull request by danilyef**
…erRerank

 added "device" parameter to SentenceTransformerRerank. It allows to use "cpu", when cuda is not available.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: llama_index/postprocessor/sbert_rerank.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def __init__(
         self,
         top_n: int = 2,
         model: str = "cross-encoder/stsb-distilroberta-base",
+        device: str = "cuda",
```
Comment (hatianzhang) [inline llama_index/postprocessor/sbert_rerank.py:21]: should by default `cpu`?

Comment (danilyef) [reply llama_index/postprocessor/sbert_rerank.py:21]: Probably, users want to use `cuda`, if it's available. So I would leave it like that tbh.

Comment (hatianzhang) [reply llama_index/postprocessor/sbert_rerank.py:21]: can you do like this 
```
device = "cuda" if torch.cuda.is_available() else "cpu"
```

File: llama_index/postprocessor/sbert_rerank.py [hunk 2]
```diff
@@ -18,6 +18,7 @@ def __init__(
         self,
         top_n: int = 2,
         model: str = "cross-encoder/stsb-distilroberta-base",
+        device: str = "cuda" if torch.cuda.is_available() else "cpu",
```
Comment (hatianzhang) [inline llama_index/postprocessor/sbert_rerank.py:21]: need to `import torch` first?

Comment (danilyef) [reply llama_index/postprocessor/sbert_rerank.py:22]: Sorry, done.

**Comment by hatianzhang** [comment]
@danilyef can use 
```
from llama_index.utils import infer_torch_device
```

**Comment by danilyef** [comment]
> @danilyef can use
> 
> ```
> from llama_index.utils import infer_torch_device
> ```

Done

**Pull request merged**